server/server.js

```diff
@@ -4,6 +4,7 @@ const db = require('../database');
 const app = express();
 const cors = require('cors');
 const path = require('path');
+require('newrelic');
```

Comment:
If you move this to the tippy top you will catch any issues with the express etc as well in New Relic